torch/csrc/autograd/variable.cpp

```diff
@@ -172,7 +166,7 @@ void Variable::Impl::release_resources() {
   hooks_.clear();
 }
 
-Variable::ViewImpl::ViewImpl(Variable base, at::Tensor data, Edge gradient_edge)
+Variable::DifferentiableViewImpl::DifferentiableViewImpl(Variable base, at::Tensor data, Edge gradient_edge)
```

Comment:
What was wrong with `ViewImpl`? This is so verbose!! There's no `NondifferentiableViewImpl`, so it seems unnecessary.

Reply:
As said in the note, there are differentiable views and non-differentiable views. This Impl only applies to the differentiable ones. Having a few extra characters is much better than making the code confusing.

This Impl isn't even used in many places, so I don't think that this is a problem.